caffe2/serialize/versions.h

```diff
@@ -67,7 +67,11 @@ constexpr uint64_t kProducedFileFormatVersion = 0x3L;
 //  0x3L: (Comment missing)
 //  0x4L: (Comment missing)
 //  0x4L: (update) Added schema to function tuple. Forward-compatible change.
-constexpr uint64_t kProducedBytecodeVersion = 0x4L;
+//  0x5L: (update) Update tensor storage schema. For bytecode archive, the
+//  root key of tensor storage is updated from {index} to {the_pointer_value_the_tensor.storage}.
+//  for example: `140245072983168.storage`
+//  Forward-compatibility change. Model size reduced by up to 50%.
```

Comment:
"Model size reduced by up to 50%." this is hard to tell for sure since it's model dependent (per the analysis you made).
I would remove it.

Reply:
Remove in last commit